.travis.yml

```diff
@@ -2,21 +2,27 @@ language: php
 
 sudo: false
 
-php: [5.3, 5.4, 5.5, 5.6, nightly, hhvm]
+php: [5.3, 5.4, 5.5, 5.6, 7.0, hhvm]
 
 matrix:
   fast_finish: true
+  include:
+    - php: 5.3
+      env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
   allow_failures:
-    - php: nightly
+    - php: 7.0
 
 cache:
   directories:
     - $HOME/.composer/cache/files
 
+before_install:
+  - composer self-update
+
 install:
-  - composer install
+  - composer update $COMPOSER_FLAGS
 
 script: phpunit -v --coverage-clover=coverage.clover
 
 after_script:
-  - if [ "nightly" != "$TRAVIS_PHP_VERSION" ]; then wget https://scrutinizer-ci.com/ocular.phar && php ocular.phar code-coverage:upload --format=php-clover coverage.clover; fi
+  - if [[ "7.0" != "$TRAVIS_PHP_VERSION" && "$TRAVIS_PHP_VERSION" != "hhvm" ]]; then wget https://scrutinizer-ci.com/ocular.phar && php ocular.phar code-coverage:upload --format=php-clover coverage.clover; fi
```

Comment:
recent PHPUnit versions are not able to generate code coverage on the old HHVM 3.5 (which is still used by Travis), so I avoid notifying Scrutinizer of a missing coverage.